powerquery-docs/connectors/solarwinds-service-desk.md

```diff
@@ -0,0 +1,64 @@
+# SolarWinds Service Desk (Beta)
```

Comment:
Above the title of the article, you'll need to add the following metadata:

\---
title: Solarwinds Service Desktop (Beta)
description: Provides basic information, prerequisites, and instructions on how to connect to the Solarwinds Service Desktop connector.
author: \<your github alias>
ms.author: bezhan
ms.topic: conceptual
ms.date: \<the date the content was written - ex: 4/14/2023>
\---

You'll need to fill in the information in <  > brackets above. Be sure to include the three dashes "---" at the beginning and end of the metadata. You can also change the description: metadata to more closely match the actual content of the article if you want.